### PR TITLE
Version 4.8.2 - substrate 0.9.31

### PR DESCRIPTION
**Pull request by rakanalh**
This is a PR that should precede #146 to be able to tag this 4.8.2 release prior to merging the 0.9.33 PR.

- [x] Change version to 4.8.2
- [x] Update spec and transaction versions
  - [x] Update cere-dev
  - [x] Update cere

### CODE REVIEW

File: runtime/cere/src/lib.rs [hunk 1]
```diff
@@ -123,10 +123,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 48009,
+	spec_version: 48010,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 6,
+	transaction_version: 7,
```
Comment (rakanalh) [inline runtime/cere/src/lib.rs:129]: Cere: increment transaction version

File: runtime/cere-dev/src/lib.rs [hunk 2]
```diff
@@ -129,10 +129,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 48015,
+	spec_version: 48016,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 6,
+	transaction_version: 7,
```
Comment (rakanalh) [inline runtime/cere-dev/src/lib.rs:135]: Cere-dev: increment transaction version

File: runtime/cere-dev/src/lib.rs [hunk 3]
```diff
@@ -129,10 +129,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 48015,
+	spec_version: 48016,
```
Comment (rakanalh) [inline runtime/cere-dev/src/lib.rs:132]: Cere-dev: increment spec version

**Review by aie0 — APPROVED** [review]

**Comment by rakanalh** [comment]
@MRamanenkau I am aware of gitflow... could you elaborate on what i am missing?

**Pull request closed without merging**